enclave_wrangler/main.py

```diff
@@ -51,13 +51,18 @@ def run(input_csv_folder_path):
     if DEBUG:
         log_debug_info()
 
+    # Read data
+    code_sets_df = pd.read_csv(os.path.join(input_csv_folder_path, 'code_sets.csv')).fillna('')
+    # For some reason, was being read with .0's at the end.
+    code_sets_df['enclave_codeset_id'] = pd.to_numeric(code_sets_df['enclave_codeset_id'], errors='coerce')\
```

Comment:
@stephanieshong This fixed the `.0` problem. 

Also, I noticed that some of the rows in `data/palantir-three-file/Lisa3/2022.03.02/output/code_sets.csv` previously had this problem as well. When I ran the script, it fixed all rows where this was previously an issue.

Also, you and I were confused where, when we were running the script during our meeting, it 2 rows were being updated instead of 1. From what I could ascertain, the reason that the row with `1000000181` was being updated in addition to `1000000273` (our test case) was because `1000000181` was the only one that didn't have a `.0` on it already. The other rows did; so they were not changed. I don't know if I'm explaining this very well, but either way, it is fixed now.

Oh, also, I found [diffMerge](https://sourcegear.com/diffmerge/) to be useful here to look at exactly which cells were changed. I believe that @Sigfried uses this or something similar.
<img width="1792" alt="diff" src="https://user-images.githubusercontent.com/13045020/158520539-f9068c52-6ca8-4e59-8bc3-95714a579595.png">

Reply:
the other rows would have been empty, since the code to add the enclave_codset_id value is newly added code. Therefore, only the rows where I was testing the archived concept would have contained the enclave_codeset_id value. This is really strange and does not explain why this would be the case

Reply:
Hmm... Maybe you're right when it comes to when you started your branch, but this diff is comparing the last commit you gave me on your branch, with the commit I made after I ran it 1 time. Either way, I suppose as long as we make sure the end result is good, that's what matters most.